libraries/ms-common/inc/pwm.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+// PWM Library
+//
+// Usage Notes:
+// - Module must be initialized before use.
+// - Use pwm_set_dc unless you have reason not to.
+// - pwm_set_pulse affords resolution < 1% of the duty cycle.
+// - For stm32f0xx GPIO pins must use the correct GPIO_ALTFN to utilize the PWM.
```

Comment:
Do we need a function to actually enable/disable certain output pins? It doesn't clash if we try to use the IO for something else while the timer output is enabled?